highlighting/lezer-grammars/level10.grammar

```diff
@@ -46,8 +46,8 @@ Else { elses+ }
 
 Condition { 
     EqualityCheck { Text is+ (String | Expression | pressed+) } |
-    InListCheck { Text ins+  Text } |
-    NotInListCheck { Text not_in+  Text }
+    InListCheck { Text ins+ Text } |
+    NotInListCheck { Text not+ ins+ Text }
```

Comment:
Hi Boryana! So cool that you're looking at the highlighter a little bit! If you have any comments with respect to that, you can let me know. 

With respect to this change, @Felienne  and I have discussed before wether having a keyword for `not` and for `in`, but it turns out that chinese actually uses just one symbol for it, so this would fail in that language. Therefore, we think is better to keep keywords under a single name, even if they have several words, since many of the languages already have spaces for many of the keywords! Does this make sense to you? What do you think?